src/pages/MainPage/Components/Search/Search.jsx

```diff
@@ -1,34 +1,34 @@
-import React, { useState } from "react";
-import { faSearch } from "@fortawesome/free-solid-svg-icons";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { useState } from 'react';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-import FilterResults from "../FilterResults/FilterResults";
-import "./style.sass";
+import FilterResults from '../FilterResults/FilterResults';
+import './style.sass';
 /*eslint-disable*/
 function Search({ test }) {
   const [areResultsActive, setAreResultsActive] = useState(false);
-  const [searchTerm, setSearchTerm] = useState("");
+  const [searchTerm, setSearchTerm] = useState('');
 
   function handleChange(event) {
     setSearchTerm(event.target.value);
   }
 
   function onClickHandler() {
-    test(searchTerm);
+    test(searchTerm.split(' ').join('_'));
 
     setAreResultsActive(true);
   }
 
   return (
     <div>
-      <div className="search-box">
-        <button className="btn-search" onClick={onClickHandler}>
+      <div className='search-box'>
+        <button className='btn-search' onClick={onClickHandler}>
           <FontAwesomeIcon icon={faSearch} />
         </button>
         <input
-          type="text"
-          className="input-search"
-          placeholder="Type to Search..."
+          type='text'
+          className='input-search'
+          placeholder='Type to Search...'
           onChange={handleChange}
         />
       </div>
```

Comment:
I suppose that filter should be available for all beers not for searched only

Reply:
Maybe, but they want it in the task

![image](https://user-images.githubusercontent.com/24357052/136836862-b3389680-b502-4336-a1c1-94fd184bd19c.png)

Reply:
You are right